From f5800718779a4f592141c228f31af82345543754 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Fri, 7 Jun 2013 13:07:47 +0800
Subject: [PATCH 206/310] ENGR00264509 fix app shortcut icon out of screen
 issue

The launcher border is out of screen, if put a app shortcut
at the border, half of the icon is cut

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 .../apps/Launcher2/res/values-sw720dp/dimens.xml     | 20 ++++++++++++++++++++
 .../apps/Launcher2/res/values-sw720dp/dimens.xml     | 20 ++++++++++++++++++++
 2 files changed, 40 insertions(+)
 create mode 100644 hdmidongle_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
 create mode 100644 sabreauto_6q/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml

diff --git a/hdmidongle_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml b/hdmidongle_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
new file mode 100644
index 0000000..d55a391
--- /dev/null
+++ b/hdmidongle_6dq/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2011 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources>
+    <dimen name="workspace_width_gap_land">32dp</dimen>
+</resources>
+
diff --git a/sabreauto_6q/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml b/sabreauto_6q/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
new file mode 100644
index 0000000..d55a391
--- /dev/null
+++ b/sabreauto_6q/overlay/packages/apps/Launcher2/res/values-sw720dp/dimens.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2011 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources>
+    <dimen name="workspace_width_gap_land">32dp</dimen>
+</resources>
+
-- 
1.8.0

